x-pack/test/functional/page_objects/observability_log_explorer.ts

```diff
@@ -260,6 +260,25 @@ export function ObservabilityLogsExplorerPageObject({
       );
     },
 
+    async navigateToDatasetQuality({
+      pageState,
+    }: {
+      pageState?: logExplorerUrlSchemaV1.UrlSchema;
```

Comment:
**nit**: navigating to the datasetQuality page, there is a dedicated urlSchema for this page, while the logsExplorer one is different, you can find the right urlSchema for the dataset quality on the same folder of this one.

Reply:
Additionally, should this helper go into the dataset quality page object as a `navigateTo` method to keep is concerned to that page object features?